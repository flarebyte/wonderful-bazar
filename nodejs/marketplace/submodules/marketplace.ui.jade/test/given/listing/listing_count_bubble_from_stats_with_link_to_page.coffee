given = 
  everything:
    title: "everything"
    pageKey: "webpage"
    statKey: "numberOfComments"
    item:
      webpage: "http://flarebyte.com"
      _stats_:
        numberOfComments: 12
  no_item:
    title: "no_item"
    pageKey: "webpage"
    statKey: "numberOfComments"
  item_is_null:
    title: "items_is_null"
    pageKey: "webpage"
    statKey: "numberOfComments"
    item: null
  stats_is_null:
    title: "stats_is_null"
    pageKey: "webpage"
    statKey: "numberOfComments"
    item:
      webpage: "http://flarebyte.com"
      _stats_: null
  no_statKey_value:
    title: "no_statKey_value"
    pageKey: "webpage"
    statKey: "numberOfComments"
    item:
      webpage: "http://flarebyte.com"
      _stats_:
        numberOfReads: 12

  no_pageKey_value:
    title: "no_pageKey_value"
    pageKey: "webpage"
    statKey: "numberOfComments"
    item:
      web: "http://flarebyte.com"
      _stats_:
        numberOfReads: 12

module.exports=given
