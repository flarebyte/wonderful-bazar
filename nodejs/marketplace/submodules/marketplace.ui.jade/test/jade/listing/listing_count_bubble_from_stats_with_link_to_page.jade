include mixins/listing
ul
  mixin listing_count_bubble_from_stats_with_link_to_page(everything.title, everything.item, everything.pageKey, everything.statKey)
  mixin listing_count_bubble_from_stats_with_link_to_page(no_item.title, no_item.item, no_item.pageKey, no_item.statKey)
  mixin listing_count_bubble_from_stats_with_link_to_page(item_is_null.title, item_is_null.item, item_is_null.pageKey, item_is_null.statKey)
  mixin listing_count_bubble_from_stats_with_link_to_page(stats_is_null.title, stats_is_null.item, stats_is_null.pageKey, stats_is_null.statKey)
  mixin listing_count_bubble_from_stats_with_link_to_page(no_statKey_value.title, no_statKey_value.item, no_statKey_value.pageKey, no_statKey_value.statKey)
  mixin listing_count_bubble_from_stats_with_link_to_page(no_pageKey_value.title, no_pageKey_value.item, no_pageKey_value.pageKey, no_pageKey_value.statKey)
