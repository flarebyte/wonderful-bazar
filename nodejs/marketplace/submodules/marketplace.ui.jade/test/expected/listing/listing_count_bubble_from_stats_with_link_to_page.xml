<ul><li><a href="http://flarebyte.com">everything<span class="ui-li-count">12</span></a></li><li><a href="http://flarebyte.com">no_statKey_value</a></li><li><a href="undefined">no_pageKey_value</a></li></ul>